docs/blueprints/EP022.rst

```diff
@@ -0,0 +1,328 @@
+:EP: 22
+:Title: Kytos Pathfinder Filter Paths by Metadata
+:Authors:
+    - David Ramirez <drami102@fiu.edu>
+    - Marvin Torres <mtorr068@fiu.edu>
+:Created: 2020-10-05
+:Kytos-Version:
+:Status: Draft
+
```

Comment:
To build the blueprints table and its navigation properly, we have a new standard in EP000 for the title as follows: 

```suggestion
***************************************************
EP022 - Kytos Pathfinder Filter Paths by Metadata
***************************************************

```

Reply:
Thank you for the heads up! I will commit your suggestion.